core/unit_test/TestOPENMP_Category.hpp

```diff
@@ -12,6 +12,7 @@
 #ifndef CABANA_TEST_OPENMP_CATEGORY_HPP
 #define CABANA_TEST_OPENMP_CATEGORY_HPP
 
+#define TEST_HOST
```

Comment:
This is not the place to put `TEST_HOST` - add something to the unit test that checks the backends and triggers the right type. These files are specific to a more general portable unit test strategy.

Reply:
I thought this was cleaner (thinking maybe this would come up in other places), but I can do this in the FFT unit test

Reply:
Removed